src/cryptojwt/key_bundle.py

```diff
@@ -153,7 +153,7 @@ def ec_init(spec):
 class KeyBundle:
     """The Key Bundle"""
 
-    def __init__(self, keys=None, source="", cache_time=300, verify_ssl=True,
```

Comment:
If we drop `verify_ssl` here, should we also drop it from `KeyBundle`?

Reply:
Yes ! It should be dropped everywhere. Backward compatibility have made me keep it in `KeyBundle` so far.